practitioner-guides/contributor-sustainability.md

```diff
@@ -7,13 +7,17 @@ Primary metrics:
+Contributor sustainability is an important part of assessing whether an open source project and community have enough contributors for the project to be sustained over the long-term, so contributor sustainability has a large impact on overall project sustainability. There are a lot of projects with a single maintainer (see [xkcd Dependency](https://xkcd.com/2347/)), and many projects struggle to find enough people to actively participate in their projects and continue to maintain them over the long term (Egbahl 2016). Avelino et al. (2019) found that even popular projects may become permanently abandoned and fail to recover from losing too many key contributors.
+
+The reality is that there are a lot of open source projects and not enough contributors, so maintainers are in desperate need for help across the various types of contributions needed to have a successful and sustainable open source project. However, it's important to think about how each new contribution will create additional work for maintainers to review and accept it, which can be mitigated by having appropriate automation, processes, and documentation to help contributors become successful, but eventually, some of these contributors will need to become maintainers to handle the additional workload from new contributors (Eghbal 2020). 
+
+If there are not enough contributors and maintainers to sustain a project, this increases the risks that the project will fail, which creates a variety of often significant challenges for the users and other projects that depend on it.
 
 # Step 1: Identify Trends
 There are a number of things that can impact contributor sustainability. By starting with [Bus Factor](https://chaoss.community/?p=3944), you can assess the risk to the project if key contributors / maintainers decided to leave, but it’s also important to look at other trends related to [Contributors](https://chaoss.community/?p=3467) along with the [Types of Contributions](https://chaoss.community/?p=3432) that people are making to identify potential contributor risks that might impact the overall sustainability of your project.
 
 ## Bus Factor
```

Comment:
Do you want to just change Bus Factor in this PR now that the change has been made?

Reply:
Not in this PR, since it's fundamentally a different change. Also Precious has submitted a PR to fix that - we'll see if that works after I merge this one.